trade/src/main/proto/musigrpc/rpc.proto

```
@@ -0,0 +1,150 @@
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+syntax = "proto3";
+
+package musigrpc;
```

Buf (1.54.0)

Files with package "musigrpc" must be within a directory "musigrpc" relative to root but were in directory "trade/src/main/proto/musigrpc".

(PACKAGE_DIRECTORY_MATCH)